src/test/scala/mesosphere/marathon/core/leadership/AlwaysElectedLeadershipModule.scala

```diff
@@ -1,21 +1,27 @@
 package mesosphere.marathon.core.leadership
 
 import akka.actor.{ ActorRefFactory, ActorRef, Props }
+import com.twitter.common.zookeeper.ZooKeeperClient
+import mesosphere.marathon.LeadershipAbdication
 import mesosphere.marathon.core.base.{ ActorsModule, ShutdownHooks }
+import mesosphere.marathon.test.Mockito
 
 /**
   * Provides an implementation of the [[LeadershipModule]] which assumes that we are always the leader.
   *
   * This simplifies tests.
   */
-object AlwaysElectedLeadershipModule {
+object AlwaysElectedLeadershipModule extends Mockito {
   def apply(shutdownHooks: ShutdownHooks): LeadershipModule = {
     val actorsModule = new ActorsModule(shutdownHooks)
-    new AlwaysElectedLeadershipModule(actorsModule.actorRefFactory)
+    val zk = mock[ZooKeeperClient]
```

Comment:
This is test code which is used to stub the LeadershipModule. So it would be even better if we avoided starting nested actors which are unused. Unfortunately, the general pattern of using "startWhenLeader" currently is to do it in the module constructors (can we change that easily?) which makes it hard to prevent. If you have a simple idea...